src/prefect/cli/create.py

```diff
@@ -19,12 +19,12 @@ def create():
 
     \b
     Examples:
-        $ prefect create project "My Project"
-        My Project created
+        $ prefect create project "Hello, World!"
+        Hello, World! created
 
     \b
-        $ prefect create project My-Project --description "My description"
-        My-Project created
+        $ prefect create project "Hello, World!" --description "My description"
```

Comment:
To make the presence/absence of quotes for CLI things more transparent, should we default to a name that wouldn't need to be quoted? Perhaps `hello-world` or something similar?

No strong thoughts, happy either way.

Reply:
I think this is actually a nice little extra documentation of "you can use names with spaces and special characters but have to quote them"

Reply:
Makes sense to me!